hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java

```diff
@@ -257,7 +257,7 @@ void openInfo(boolean refreshLocatedBlocks) throws IOException {
         // locations will not be available with NN for getting the length. Lets
         // retry for 3 times to get the length.
         if (lastBlockBeingWrittenLength == -1) {
-          DFSClient.LOG.warn("Last block locations not available. "
+          DFSClient.LOG.warn("Last block locations " + getCurrentBlock() + " not available. "
```

Comment:
Also add `src` will be more helpful IMO. Thanks.

Reply:
Thanks @Hexiaoqiao for your review , just have updated `src` info in the log, but i have not idea on the log test cases ,Could you give me some advice ?